src/eh1.rs

```diff
@@ -8,8 +8,10 @@ mod error;
 pub use crate::eh1::error::MockError;
 
 pub mod delay;
+pub mod digital;
 pub mod i2c;
-pub mod pin;
+#[deprecated]
+pub use digital as pin;
```

Comment:
Does this show up as a deprecation warning when importing `pin`?

If yes, could we add a message to the deprecation warning, indicating what to use instead? https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute

Reply:
I agree. We're still at 0.x, so a breaking change is fine.